cmd/trace-agent/config/config.go

```diff
@@ -687,3 +688,14 @@ func setMaxMemCPU(c *config.AgentConfig, isContainerized bool) {
 		c.MaxMemory = 0
 	}
 }
+
+func isObsPipelineEnabled() (bool, string) {
+	if coreconfig.Datadog.GetBool("observability_pipelines_worker.traces.enabled") {
+		return true, "observability_pipelines_worker"
+	}
+	// Vector config is deprecated in favor of observability_pipelines_worker.
```

Comment:
I'm not really sure that we will be "deprecating" the `vector` name for these options since we want to still support receiving data from the Datadog Agent in Vector. I think it's more just that we want an alias for the options for Observability Pipelines users.

Reply:
I see. Ill remove the comments in that case!